src/js/jsx/sections/style/Shadow.jsx

```diff
@@ -445,10 +445,19 @@ define(function (require, exports) {
             if (layers.isEmpty()) {
                 return null;
             }
-
-            // Group into arrays of dropShadows, by position in each layer
-            var dropShadowGroups = collection.zip(collection.pluck(layers, "dropShadows")),
-                dropShadowList = dropShadowGroups.map(function (dropShadows, index) {
+                
+            var shadowCountsUnmatch = layers.size > 1 &&
+                    Immutable.Set(layers.map(function (l) { return l.dropShadows.size; })).size !== 1,
```

Comment:
agh... I mean... there has to be a simpler way for this... or at least can be broken into couple lines to be more readable.

Maybe `layers.some(function (l) { return l.dropShadows.size !== layers.first().dropShadows.size; })`